### PR TITLE
Homework 11.1

### PR DESCRIPTION
**Pull request by AnkaMikola**
Лінтер показує помилку: 

homework_11.py:28:1: S101 Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.

але як виконати це завдання без "assert"?..

### CODE REVIEW

File: Homeworks/homework_11.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        actual_result = ' - '.join(actual_result.split(' - ')[1:])
+        assert actual_result[:-1] == expected_result
+
+
+@pytest.mark.parametrize('username, status', [
+    ('Alice', 'success'),
+    ('Mykola', 'expired'),
+    ('Taras', 'failed'),
+])
+def test_log_event(username: str, status: str):
```
Comment (antohaUa) [inline Homeworks/homework_11.py:36]: Please split this test case into several test cases

File: Homeworks/hw9/homework09.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
+"""
```
Comment (antohaUa) [inline Homeworks/hw9/homework09.py:1]: Remove this file

**Review by antohaUa — CHANGES_REQUESTED** [review]

**Comment by AnkaMikola** [comment]
https://github.com/AnkaMikola/AQA_Hillel_git/pull/35

File: Homeworks/homework_11.py (hunk 1 above)
Comment (AnkaMikola) [reply Homeworks/homework_11.py:36]: https://github.com/AnkaMikola/AQA_Hillel_git/pull/35

**Pull request closed without merging**